json-endpoints/jetpack/class.jetpack-json-api-plugins-endpoint.php

```diff
@@ -129,6 +131,9 @@ protected function format_plugin( $plugin_file, $plugin_data ) {
 	}
 
 	protected function get_plugins() {
+		// Do the admin_init action in order to capture plugin action links.
+		// See get_plugin_action_links()
+		do_action( 'admin_init' );
```

Comment:
Not much I'm afraid, but you can take a look at the result [here](http://developer.jetpack.com/). The parser itself lives on WordPress.com, and you can check one of the most updates in r130226-wpcom.